tools/apiview/parsers/cpp-api-parser/ci.yml

```diff
@@ -76,14 +76,8 @@ stages:
             # This step is idempotent and can be run multiple times in cases of
             # failure and partial execution.
             retryCountOnTaskFailure: 3
-          # - pwsh: |
-          #     ls ${env:VCPKG_ROOT}
-          #     ./vcpkg/vcpkg.exe install atlmfc:x64-windows
-          #   displayName: Install atlmfc.
-          #   workingDirectory: '$(Build.SourcesDirectory)'
-          #   condition: succeeded()
           - pwsh: |
-              cmake.exe -G "Visual Studio 17 2022" -DCMAKE_CXX_STANDARD:STRING="20" -DCMAKE_TOOLCHAIN_FILE:STRING=${env:VCPKG_ROOT}/scripts/buildsystems/vcpkg.cmake -DVCPKG_BUILD_TYPE:STRING="release" -DVCPKG_OVERLAY_TRIPLETS:STRING="../vcpkg-triplets" -DMSVC_RUNTIME_LIBRARY:STRING="MultiThreadedDebug" -DVCPKG_TARGET_TRIPLET:STRING="x64-windows-static-release" ..
+              cmake.exe -G "Visual Studio 16 2019" -DCMAKE_CXX_STANDARD:STRING="20" -DCMAKE_TOOLCHAIN_FILE:STRING=${env:VCPKG_ROOT}/scripts/buildsystems/vcpkg.cmake -DVCPKG_BUILD_TYPE:STRING="release" -DVCPKG_OVERLAY_TRIPLETS:STRING="../vcpkg-triplets" -DMSVC_RUNTIME_LIBRARY:STRING="MultiThreadedDebug" -DVCPKG_TARGET_TRIPLET:STRING="x64-windows-static-release" ..
```

Comment:
I think the issue is that for vscode 2022, we'll need to install ATL/MFC. It's no longer automatically included.

Reply:
If it's not included now, it may not be included in the future. Eventually 2019 will be extinct. Seems like we should try to solve this problem now, or at least see if it's easy to script+takes a short time.

Reply:
Actually there are several problems here:
1) The current RTM version of VS 2022 has a number of issues (working with the VS and 1ES teams to resolve).
2) We need a new APiView based off of clang 17, not clang15 because of changes to STL in VS 2022.

We still don't understand the afxres.h file being missing - the visual studio team says that it was *not* removed from the ATLMFC workload.

As a temporary workaround, this might work however.